packages/skeleton-react/src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,68 @@
+import { useEffect } from "react";
+import { ProgressProps } from "./types";
+
+export default function Progress({
```

Comment:
We can add typing to indicate this is a React function component. See the Avatar component as reference

```ts
export const Avatar: React.FC<AvatarProps> = ()
```

Might be worth checking if this is better to use this explicitly or not.